File: caesars-cipher.js
function rot13(str) {
  const shift = 13;
  arr = str.toUpperCase().split("");
  arr = arr.map(x => {
    let code = x.charCodeAt(0);
    if (code >= 65 && code <= 90) {
      code += shift;
      if (code > 90) {
        code -= 26;
      }
    }
    return code;
  });
  return String.fromCharCode.apply(null, arr);
}

// Change the inputs below to test
answer = rot13("SERR PBQR PNZC");
console.log(answer);

File: caesars_cipher.py
# use ord() and chr() to convert


def caesar(str, shift=13):
    new_word = ''
    str = str.upper()
    for char in str:
        if char.isalpha():
            char = ord(char) + shift
            if char > 90:
                char -= 26
            char = chr(char)
        new_word += char
    print(new_word)
    return new_word


caesar("SERR PBQR PNZC")
